sentry_sdk/integrations/starlette.py

```diff
@@ -447,26 +447,16 @@ async def extract_request_info(self):
             if _should_send_default_pii():
                 request_info["cookies"] = self.cookies()
 
-            if not request_body_within_bounds(client, content_length):
-                data = AnnotatedValue(
-                    "",
-                    {
-                        "rem": [["!config", "x", 0, content_length]],
-                        "len": content_length,
-                    },
-                )
+            if not content_length or not request_body_within_bounds(
```

Comment:
if `content_length` is not there doesn't mean that we truncated it.
This should be a separate case. We should just not do anything in that case.
So
```python
if content_length:
   if not request_body_bla():
      data = AnnotatedValue
    else:
       do stuff
```

Reply:
done.